core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java

```diff
@@ -41,74 +44,18 @@
-	
-	private static final String FUSE_621_R0_BOM_VERSION = "6.2.1.redhat-084";
-	private static final String FUSE_621_R1_BOM_VERSION = "6.2.1.redhat-090";
-	private static final String FUSE_621_R2_BOM_VERSION = "6.2.1.redhat-107";
-	private static final String FUSE_621_R3_BOM_VERSION = "6.2.1.redhat-117";
-	private static final String FUSE_621_R4_BOM_VERSION = "6.2.1.redhat-159";
-	private static final String FUSE_621_R5_BOM_VERSION = "6.2.1.redhat-169";
-	private static final String FUSE_621_R6_BOM_VERSION = "6.2.1.redhat-177";
-	private static final String FUSE_621_R7_BOM_VERSION = "6.2.1.redhat-186";
-	private static final String FUSE_621_R8_BOM_VERSION = "6.2.1.redhat-195";
-	private static final String FUSE_621_R9_BOM_VERSION = "6.2.1.redhat-216";
-	
-	private static final String FUSE_63_R0_CAMEL_VERSION = "2.17.0.redhat-630187";
-	private static final String FUSE_63_R1_CAMEL_VERSION = "2.17.0.redhat-630224";
-	private static final String FUSE_63_R2_CAMEL_VERSION = "2.17.0.redhat-630254";
-	private static final String FUSE_63_R3_CAMEL_VERSION = "2.17.0.redhat-630262";
-	public static final String FUSE_63_R4_CAMEL_VERSION = "2.17.0.redhat-630283";
-	public static final String FUSE_63_R5_CAMEL_VERSION = "2.17.0.redhat-630310";
-
-	private static final String FUSE_63_R0_BOM_VERSION = "6.3.0.redhat-187";
-	private static final String FUSE_63_R1_BOM_VERSION = "6.3.0.redhat-224";
-	private static final String FUSE_63_R2_BOM_VERSION = "6.3.0.redhat-254";
-	private static final String FUSE_63_R3_BOM_VERSION = "6.3.0.redhat-262";
-	static final String FUSE_63_R4_BOM_VERSION = "6.3.0.redhat-283";
-	static final String FUSE_63_R5_BOM_VERSION = "6.3.0.redhat-310";
-	
-	public static final String FIS_20_R1_CAMEL_VERSION = "2.18.1.redhat-000012";
-	public static final String FIS_20_R2_CAMEL_VERSION = "2.18.1.redhat-000015";
-	public static final String FIS_20_R3_CAMEL_VERSION = "2.18.1.redhat-000021";
-	
-	public static final String DEFAULT_CAMEL_VERSION = FIS_20_R3_CAMEL_VERSION;
+	public static final String DEFAULT_CAMEL_VERSION = CamelForFIS6ToBomMapper.FIS_20_R3_CAMEL_VERSION;
 	
 	public static final String CAMEL_SPRING_BOOT_STARTER = "camel-spring-boot-starter";
 	public static final String CAMEL_WILDFLY = "org.wildfly.camel";
 	
 	public static final String CAMEL_VERSION_LATEST_COMMUNITY = "2.20.2";
-	public static final String CAMEL_VERSION_LATEST_PRODUCTIZED_62 = FUSE_621_R9_CAMEL_VERSION;
-	public static final String CAMEL_VERSION_LATEST_PRODUCTIZED_63 = FUSE_63_R5_CAMEL_VERSION;
-	public static final String CAMEL_VERSION_LATEST_FIS_20 = FIS_20_R3_CAMEL_VERSION;
+	public static final String CAMEL_VERSION_LATEST_PRODUCTIZED_62 = CamelForFuse6ToBomMapper.FUSE_621_R9_CAMEL_VERSION;
+	public static final String CAMEL_VERSION_LATEST_PRODUCTIZED_63 = CamelForFuse6ToBomMapper.FUSE_63_R5_CAMEL_VERSION;
+	public static final String CAMEL_VERSION_LATEST_FIS_20 = CamelForFIS6ToBomMapper.FIS_20_R3_CAMEL_VERSION;
```

Comment:
Isn't it FIS 2.0 ? Why did u choose FIS6 here?

Reply:
no big reasons except that it was to make clear opposition with "Fuse 7 On OpenShift". FIS 2.0 seems not clear enough when i wrote this classname but maybe better to keep FIS 2.0 to better match other naming.